Maple2.Server.Game/Packets/NpcControlPacket.cs

```diff
@@ -33,29 +34,52 @@ public static class NpcControlPacket {
         buffer.WriteByte(2);
 
         buffer.Write<Vector3S>(npc.Position);
-        buffer.WriteShort((short) (npc.Rotation.Z * 10));
-        buffer.Write<Vector3S>(npc.Velocity.Rotate(npc.Rotation));
-        buffer.WriteShort((short) (sequenceSpeed * 100));
+        buffer.WriteShort((short) (npc.Transform.RotationAnglesDegrees.Z * 10));
+        buffer.Write<Vector3S>(npc.MovementState.Velocity);
+        //buffer.Write<Vector3S>(new Vector3(0, 0, 0));
```

Comment:
probably can cleanup some commented code in here

Reply:
Woops, I did a pass over for these too but missed this apparently lol